WPMedia/arc-themes-components
# Themes 1440 twitter icon

## Ticket

- [THEMES-1440](https://arcpublishing.atlassian.net/browse/THEMES-1440)

## Description

Update on Twitter Icon

## Acceptance Criteria

[_copy from ticket_](https://www.figma.com/file/NBjMF1v4mzwv9jkn7NEq9V/Themes-Block-Styling-2.0?type=design&node-id=10811%3A147806&mode=design&t=jG5oCFMUMr3b84Ro-1)

## Test Steps

1. Checkout branch - `git checkout THEMES-1440-Twitter-icon`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Check out the Twitter icon in storybook

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1440]: https://arcpublishing.atlassian.net/browse/THEMES-1440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
